validate/validate_test.go

```diff
@@ -0,0 +1,32 @@
+package validate
+
+import (
+	"strings"
+	"testing"
+
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func checkErrors(t *testing.T, title string, msgs []string, valid bool) {
+	if valid && len(msgs) > 0 {
+		t.Fatalf("%s: expected not to get error, but get %d errors:\n%s", title, len(msgs), strings.Join(msgs, "\n"))
+	} else if !valid && len(msgs) == 0 {
+		t.Fatalf("%s: expected to get error, but actually not", title)
+	}
+}
+
+func TestCheckSemVer(t *testing.T) {
+	cases := []struct {
+		val      string
+		expected bool
+	}{
+		{rspec.Version, true},
+		{"0.0.1", false},
```

Comment:
On Mon, Sep 05, 2016 at 03:38:57AM -0700, David Liang wrote:
> > +		{"0.0.1", false},
> 
> Currently only support rspec.Version.

So maybe leave a comment on this line that we're confirming a
shortcoming of the current test suite with this entry.